packages/zowe-explorer/src/trees/dataset/ZoweDatasetNode.ts

```diff
@@ -834,6 +834,11 @@ export class ZoweDatasetNode extends ZoweTreeNode implements IZoweDatasetTreeNod
 
             await this.listMembers(responses, { attributes: true, start, maxLength: start ? limit + 1 : limit });
         } catch (err) {
+            if (err instanceof imperative.ImperativeError) {
+                if (Number(err.errorCode) === imperative.RestConstants.HTTP_STATUS_404) {
+                    return { items: [] };
+                }
+            }
```

Comment:
Thoughts on reducing this code a bit? This would minimize code complexity & remove the need for the nested `if`

```suggestion
            if (err instanceof imperative.ImperativeError && Number(err.errorCode) === imperative.RestConstants.HTTP_STATUS_404) {
                    return { items: [] };
            }
```